lib/tasks/foreman_dlm_tasks.rake

```diff
@@ -13,7 +15,7 @@ namespace :test do
 end
 
 namespace :foreman_dlm do
-  task :rubocop do
+  task rubocop: :environment do
```

Comment:
Like in foreman_rescue, I think you can clean up most of this because it was for Jenkins.